roles/aap_setup_install/tasks/main.yml

```diff
@@ -75,11 +75,10 @@
       until: __aap_setup_inst_result_ah.status == 200
       retries: 90
       delay: 10
-      when: "'automationhub' in aap_setup_prep_inv_nodes"
+      when: "'automationhub' in aap_setup_prep_inv_nodes and aap_postinstall_validation|bool == true"
   when: >
     aap_setup_inst_force
     or ('automationcontroller' in aap_setup_prep_inv_nodes
         and __aap_setup_inst_ctl_check.status != 200)
     or ('automationhub' in aap_setup_prep_inv_nodes
         and __aap_setup_inst_ah_check.status != 200)
-...
```

Comment:
we require ... for ending yaml files, please add that back

Reply:
@djdanielsson updated!